Sources/OutputTarget.swift

```diff
@@ -32,7 +32,7 @@ import Darwin.C
 
 private func getEnvValue(key: String) -> String? {
     let value = getenv(key)
-    return value != nil ? String.fromCString(value) : nil
+    return value != nil ? String(cString: value!) : nil
```

Comment:
The force unwrapping will retrieve the real value from an optional value, without considering the possibility of `nil`. It says like this when used: "here is an optional, give the value to me or let me die if there is no value in it".